webapp/src/utils/GCalendar.js

```diff
@@ -0,0 +1,44 @@
+import configs from "./config";
+
+export function addEvents(events) {
+
+    // Client ID and API key from the Developer Console
+
+    const CLIENT_ID = configs["client-id"];
+    const API_KEY = configs["api-key"];
+    const DISCOVERY_DOCS = configs["discoveryDocs"];
+
+    // Authorization scopes required by the API; multiple scopes can be
+    // included, separated by spaces.
+
+    const SCOPES = configs["scopes"];
+
+    function start() {
+        window.gapi.client.init({
+            apiKey: API_KEY,
+            clientId: CLIENT_ID,
+            discoveryDocs: DISCOVERY_DOCS,
+            scope: SCOPES
+        }).then(function () {
+            window.gapi.auth2.getAuthInstance().isSignedIn.listen(addEvents);
+            addEvents(window.gapi.auth2.getAuthInstance().isSignedIn.get());
+        }).then(function () {
+            window.gapi.auth2.getAuthInstance().signIn();
+        });
+    }
+
+    function addEvents(isSignedIn) {
```

Comment:
what does this isSignedIn variable do? It looks like it is the event array passed in